deps.edn

```diff
@@ -1,14 +1,23 @@
 {:paths   ["src"]
- :deps    {org.clojure/clojure       {:mvn/version "1.10.3"}
-           org.clojure/clojurescript {:mvn/version "1.11.4"}
+ :deps    {org.clojure/clojure       {:mvn/version "1.11.1"}
+           org.clojure/clojurescript {:mvn/version "1.11.60"}
            org.clojure/core.async    {:mvn/version "1.5.648"}}
  :aliases {:cljs   {:extra-deps  {thheller/shadow-cljs {:mvn/version "2.17.2"}
```

Comment:
I think the version of shadow-cljs has to match the clojurescript version. Can you please update all the deps at once? I am using [neil ](https://github.com/babashka/neil) for it, it's awesome.